script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -722,7 +718,7 @@ An awareness of death that surpasses that of a beast.
 -- Page 601, Offset 10246.
 -- ヤツは何の根拠もない状況で、<敵|・><に|・><は|・><自|・><分|・><を|・><殺|・><す|・><手|・><段|・><が|・><あ|・><る|・><と|・><直|・><感|・><し|・>、戦慄し、俺を脅威と認めたからこそ、一秒の迷いもなく飛び退きやがった……！
 // Ruby emphasis 敵には自分を殺す手段があると直感し
-Despite having no reason to think so, that guy had a sudden intuition that the enemy had a means to kill him, and was afraid. Since he recognized me as a threat, he leapt away without even a second's hesitation.
+Despite having no reason to think so, that guy had a %{i}sudden intuition that the enemy had a means to kill him%{/i}, and was afraid. Since he recognized me as a threat, he leapt away without even a second's hesitation.
```

Comment:
Not sure about this one, if we do keep it maybe shorten to just 
> _the enemy has a means to kill him_

Reply:
i say no italics